### PR TITLE
Root window fixes

### PR DESCRIPTION
**Pull request by rodrmoya**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/801945

This depends on https://github.com/mono/mono-addins/pull/138

### CODE REVIEW

**Comment by slluis** [comment]
@rodrmoya compatibility check is failing, due to changes in dependencies of mono-addins.

**Comment by rodrmoya** [comment]
cc @KirillOsenkov for the BinaryCompatBaseLine change

**Comment by slluis** [comment]
@monojenkins backport release-8.0-integration

File: main/build/MacOSX/BinaryCompatBaseline.txt [hunk 1]
```diff
@@ -94,6 +94,7 @@ In assembly 'Microsoft.VisualStudio.QualityTools.WebTestFramework, Version=10.0.
 In assembly 'Microsoft.VisualStudio.UwpTestHostRuntimeProvider.Deployment, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a': unable to resolve reference to 'Microsoft.Build, Version=15.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 In assembly 'Microsoft.VisualStudio.UwpTestHostRuntimeProvider.Deployment, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a': unable to resolve reference to 'Microsoft.UniversalApps.Deployment, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 In assembly 'Microsoft.VisualStudio.UwpTestHostRuntimeProvider.Deployment, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a': unable to resolve reference to 'Microsoft.VisualStudio.Shell.Interop.11.0, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+In assembly 'Mono.Cecil.Pdb, Version=0.10.0.0, Culture=neutral, PublicKeyToken=50cebf1cceb9d05e': Unable to resolve member reference 'System.String Mono.Cecil.TypeParser::ToParseable(Mono.Cecil.TypeReference)' from assembly 'Mono.Cecil, Version=0.10.0.0, Culture=neutral, PublicKeyToken=50cebf1cceb9d05e'
```
Comment (Therzok) [inline main/build/MacOSX/BinaryCompatBaseline.txt]: This is a problem.

Comment (Therzok) [reply main/build/MacOSX/BinaryCompatBaseline.txt]: Why we are picking up a cecil with a missing member?

**Comment by rodrmoya** [comment]
Closing this in favour of the release-8.0-based one

**Pull request closed without merging**